docs/usage/create.rst.inc

```diff
@@ -39,7 +39,7 @@ Exclusion options
     ``--exclude-if-present NAME``
```

Comment:
Thanks, but these edits need to go into src/borg/archiver.py --- see note ` .. IMPORTANT: this file is auto-generated from borg's built-in help, do not edit!` at the beginning of these files.